src/components/testCaseConfiguration/testCaseData/TestCaseData.tsx

```diff
@@ -68,7 +70,28 @@ const TestCaseData = () => {
     setToastOpen(open);
   };
 
-  const handleSubmit = async (values) => {};
+  const handleSubmit = async (values) => {
+    testCaseService.current
+      .shiftAllTestCaseDates(
+        measure.id,
+        parseInt(formik.values.shiftTestCaseDates)
+      )
+      .then(() => {
+        handleToast(
+          "success",
+          `Test Case Shift Dates for measure: ${measure.id} successful.`,
```

Comment:
`All Test Case dates successfully shifted` as per the ticket

Reply:
@adongare This is done too thanks!